### ISSUE TITLE
Ignore directories

### ISSUE THREAD

**Issue by calebmer** [calebmer/accelerate#3]
I have two feature requests for accelerate. Here is the first one:

Accelerate should ignore directories. Once you start to get a lot of motions it can get hard to organize, therefore some people might want to use a folder structure. E.g:

```
- 0.x.x
  |- 0.0.1-hello-world.add
  |- 0.0.1-hello-world.sub
  |- 0.0.2-lorem-ipsum.add
  |- 0.0.2-lorem-ipsum.sub
- 1.x.x
  |- 1.0.1-hello-world.add
  |- 1.0.1-hello-world.sub
  |- 1.0.2-lorem-ipsum.add
  |- 1.0.2-lorem-ipsum.sub
```

This should be a simple change, all that is required would be flattening all of the file names into a single list.

**Comment by svmnotn** [comment]
Done!. look at  a84b4da in the PR

**Comment by calebmer** [comment]
Ahh. But it's not bash 😊

**Comment by svmnotn** [comment]
I posted the wrong commit, check again

**Comment by svmnotn** [comment]
Anyways this feat is done in PR #2

**Comment by svmnotn** [comment]
Closed with a84b4da
